lib/renderer/init.js

```diff
@@ -121,7 +121,7 @@ if (preloadScript) {
   try {
     require(preloadScript)
   } catch (error) {
-    if (error.code === 'MODULE_NOT_FOUND') {
+    if (error.code === 'MODULE_NOT_FOUND' && error.message.indexOf(preloadScript) > -1) {
```

Comment:
The `error.message` variable returns "Cannot find module '/foo/bar.js'", so `===` 0 will not match. Could change it to
```
error.message.indexOf(`Cannot find module \'${preloadScript}`) === 0
```